Core/LocalizationCache.cs

```diff
@@ -7,50 +7,49 @@ public static class LocalizationCache
 	{
 		private static ConcurrentDictionary<string, object> _memoryCache = new ConcurrentDictionary<string, object>();
 		static ILocalizationCache _cache;
+
+		const string LocalizationChannel = "localization";
 		public static void Initialize(ILocalizationCache cache)
 		{
 			_cache = cache;
+			_cache.Subscribe(LocalizationChannel, OnClearKey);

```

Comment:
This is an open source lib and i don't think that most of people use redis or something like that. The interface should be as much simple as possible i think.